lib/rubocop/cop/rails/exit.rb

```diff
@@ -16,6 +16,14 @@ module Rails
       # - Usage in application code outside of the web process could result in
       # the program exiting, which could result in the code failing to run and
       # do its job.
+      #
+      # @example
+      #   # bad
+      #   exit(0)
+      #
+      #   # good
+      #   raise 'a bad error has happened'
+      #   return
```

Comment:
I think it's a bit odd to have `return` after an unconditional `raise` 😉

Reply:
Meant to be unrelated examples. Removed the `return` example.